src/lucky_web/allowed_in_tags.cr

```diff
@@ -0,0 +1,2 @@
+module LuckyWeb::AllowedInTags
```

Comment:
This needs to be included in the Int32 and Int64 structs. That can be done by adding some extensions in `src/charms`

Reply:
Ah thanks. After moving the files to the `/charms` folder I forgot a `git add`. 🕶